package.json

```diff
@@ -195,6 +215,21 @@
         "command": "one.viewer.jsonTracer",
         "title": "json tracer",
         "category": "ONE"
+      },
+      {
+        "command": "one.viewer.metadata.show",
```

Comment:
Could you remove these `Metadata Viewer`-related lines? (2)

Reply:
ok, I`ll apply that part.